test/Core.Test/OrganizationFeatures/OrganizationSubscriptionUpdate/UpdateSecretsManagerSubscriptionCommandTests.cs

```diff
@@ -443,7 +587,11 @@ await sutProvider.GetDependency<IPaymentService>().Received(1)
             )
         );
 
-        await sutProvider.GetDependency<IOrganizationService>().Received(1).ReplaceAndUpdateCacheAsync(organization);
+        await sutProvider.GetDependency<IOrganizationService>().Received(1).ReplaceAndUpdateCacheAsync(
+            Arg.Is<Organization>(org =>
+                    org.SmSeats == newSeatTotal
+            )
```

Comment:
You need to assert the other relevant properties as well (e.g. `SmServiceAccounts`, `MaxAutoscaleSmSeats` and `MaxAutoscaleSmServiceAccounts`)

Reply:
done